fusionfrontend/src/assets/i18n/de.json

```diff
@@ -42,6 +42,24 @@
         "ASSET": "Bild von Asset",
         "ASSET_SERIES": "Bild von Baureihe",
         "IMAGE": "Bild"
+      },
+      "DAYS": {
+        "MONDAY": "Montag",
+        "TUESDAY": "Dienstag",
+        "WEDNESDAY": "Mittwoch",
+        "THURSDAY": "Donnerstag",
+        "FRIDAY": "Freitag",
+        "SATURDAY": "Samstag",
+        "SUNDAY": "Sonntag"
+      },
+      "DAYS_SHORT": {
+        "MONDAY": "Mo",
+        "TUESDAY": "Di",
+        "WEDNESDAY": "Mi",
+        "THURSDAY": "Do",
+        "FRIDAY": "Fr",
+        "SATURDAY": "Sa",
+        "SUNDAY": "So"

```

Comment:
Is this not already provided by the I18N lib?

Reply:
Do you know examples? I am not aware of default translations and was not able to find some quickly on the internet..

Reply:
Probably the best approach would be to use https://day.js.org/docs/en/i18n/loading-into-browser (previously moment.js)

Reply:
due to limit of time I leave it currently as it is, but added a comment to the link